bundle/src/types.rs

```diff
@@ -85,50 +91,79 @@ pub struct BundleUploader {
 }
 
 #[derive(Debug, Serialize, Deserialize, Clone, Default, PartialEq)]
+#[cfg_attr(feature = "wasm", wasm_bindgen)]
 pub enum FileSetType {
     #[default]
     Junit,
 }
 
 #[derive(Debug, Serialize, Deserialize, Clone, Default)]
+pub struct MapType(pub HashMap<String, String>);
+
+#[cfg(feature = "wasm")]
+impl WasmDescribe for MapType {
+    fn describe() {
+        js_sys::Map::describe();
+    }
+}
+#[cfg(feature = "wasm")]
+impl IntoWasmAbi for MapType {
+    type Abi = u32;
+    fn into_abi(self) -> Self::Abi {
+        let map = js_sys::Map::new();
+        for (key, value) in self.0 {
+            map.set(&key.into(), &value.into());
+        }
+        map.into_abi()
+    }
+}
+#[cfg(feature = "wasm")]
+impl FromWasmAbi for MapType {
+    type Abi = u32;
+    unsafe fn from_abi(_js: Self::Abi) -> Self {
+        let map = HashMap::new();
+        Self(map)
+    }
+}
+impl Deref for MapType {
+    type Target = HashMap<String, String>;
+    fn deref(&self) -> &Self::Target {
+        &self.0
+    }
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone, Default)]
+#[cfg_attr(feature = "wasm", wasm_bindgen(getter_with_clone))]
 pub struct BundledFile {
     pub original_path: String,
-    pub original_path_rel: String,
+    /// Added in v0.5.33
+    pub original_path_rel: Option<String>,
     pub path: String,
+    // u128 will be supported in the next release after 0.2.95
+    #[cfg(feature = "wasm")]
+    #[wasm_bindgen(skip)]
```

Comment:
I tried patching this myself and hit a host of issues (related to transitive dependencies, not to any breaking changes). Easier and more tractable for now to just ignore this field